meta-resin-morty/recipes-core/plymouth/plymouth/0001-plymouth-systemd-append.patch

```diff
@@ -0,0 +1,116 @@
+From 86d1a1016b9ffa401c8596383183f97f53834e10 Mon Sep 17 00:00:00 2001
```

Comment:
There is an inconsistency of commit messages for these plymouth patches.

Reply:
Also, calling it "add bbappend is not descriptive in any way". Maybe something like "Migrate changes from common layer"?

Reply:
Reworded the commit message